### PR TITLE
Open graph

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: app/views/conferences/show.html.haml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+- content_for :head do
+  %meta{ property: "og:title", content: @conference.title }
+  %meta{ property: "og:site_name", content: (ENV['OSEM_NAME'] || 'OSEM') }
+  %meta{ property: "og:description", content: @conference.description }
+  %meta{ property: "og:url", content: conference_path(@conference.short_title) }
+  - @conference.picture?
```
Comment (differentreality) [inline app/views/conferences/show.html.haml]: We need an if here

Comment (ChrisBr) [inline app/views/conferences/show.html.haml]: of course we do ...

**Comment by ChrisBr** [comment]
@differentreality please review.
You can test here:
https://developers.facebook.com/tools/debug/sharing
(I already created a test event, note that it has to be accepted, otherwise the link redirects of course to the conference index page)

**Comment by differentreality** [comment]
:+1:

**Pull request merged**